i18n/en.json

```diff
@@ -685,7 +685,7 @@
   },
   {
     "id": "api.context.invalid_team_url.debug",
-    "translation": "TeamURL accessed when not valid. Team URL should not be used in api functions or those that are team independent"
+    "translation": "Team URL accessed when not valid. Team URL should not be used in API functions or those that are team independent"
```

Comment:
This is just a debug message, and it will be clear to developers what it means. I don't think it needs any updating